R/check_structure.R

```diff
@@ -278,16 +278,15 @@ check_news_protocol <- function(x) {
       ]
 
     current_link <- headings_link[grepl(version_number, headings_link)]
+    correct_link <- paste0("../", version_number, "/index.html")
 
     problems <- c(
       problems,
       sprintf(
-        "URL for version %s of the protocol is not correct in NEWS.md",
-        version_number
-      )[!identical(current_link,
-                   paste0("../",
-                          version_number,
-                          "/index.html"))]
+        "URL for version '%1$s' of the protocol should be '%2$s' in NEWS.md",
```

Comment:
```suggestion
        "URL for version '%1$s' of the protocol in NEWS.md should be '%2$s'",
```
To me it sounds a bit strange to have the `in NEWS.md` in the end of the sentence, but my English is bad, so please consider carefully or ask a second opinion...

Reply:
I think the original sentence better suggests that the change is needed in the file NEWS.md

Reply:
I personally dislike it to wait till the end of the sentence to find out that the message is about the NEWS.md file (after which I need to reread what I have to change).  ;-)  But I leave it up to you, I don't know which one is most commonly used in English.  (I remember from English course that the word order in the sentences was different from Dutch, but I don't remember the details.)

Reply:
see [92d78fd](https://github.com/inbo/protocolhelper/pull/90/commits/92d78fd73bc4a94ffc31c7553de89499cb220ec6)

Reply:
Perfect!
Eh, is this the NEWS.md of the protocol, or the general NEWS.md?  I suppose it is the NEWS.md of the protocol, and I suppose the users know that (as they don't need to make changes in the general NEWS.md)?  In this latter case the current version is fine, otherwise you may want to specify.